ncm-spma/src/main/bin/spma

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
care to comment on  https://github.com/quattor/configuration-modules-core/issues/764